src/lib/utils/wagmi.ts

```diff
@@ -1,72 +1,54 @@
-import { connectorsForWallets } from '@rainbow-me/rainbowkit'
-import {
-  argentWallet,
-  braveWallet,
-  coinbaseWallet,
-  ledgerWallet,
-  metaMaskWallet,
-  okxWallet,
-  rainbowWallet,
-  safeWallet,
-  trustWallet,
-  walletConnectWallet,
-} from '@rainbow-me/rainbowkit/wallets'
-import { Chain, http } from 'viem'
-import { createConfig } from 'wagmi'
+import { defaultWagmiConfig } from '@web3modal/wagmi/react/config'
+import { cookieStorage, createStorage, http } from 'wagmi'
 import { arbitrum, base, localhost, mainnet } from 'wagmi/chains'
 
-import { AlchemyApiKey } from '../../constants/server'
+export const projectId = process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID
+
+if (!projectId) {
+  throw new Error('Project ID is not defined')
+}
 
 const isDevelopmentEnv = process.env.NEXT_PUBLIC_VERCEL_ENV === 'development'
 const isPreviewEnv = process.env.NEXT_PUBLIC_VERCEL_ENV === 'preview'
 const shouldShowLocalHost = isDevelopmentEnv || isPreviewEnv
 
-const projectId = process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID!
-
-const connectors = connectorsForWallets(
-  [
-    {
-      groupName: 'Recommended',
-      wallets: [
-        safeWallet,
-        metaMaskWallet,
-        rainbowWallet,
-        argentWallet,
-        coinbaseWallet,
-        ledgerWallet,
-        okxWallet,
-      ],
-    },
-    {
-      groupName: 'Others',
-      wallets: [walletConnectWallet, braveWallet, trustWallet],
-    },
-  ],
-  {
-    appName: 'Index App',
-    projectId,
-  },
-)
+// Create a metadata object
+const metadata = {
+  name: 'index-coop',
+  description: 'AppKit Example',
+  url: 'https://reown.com/appkit', // origin must match your domain & subdomain
+  icons: ['https://assets.reown.com/reown-profile-pic.png'],
```

Comment:
I believe this gets overridden in the provider, but could you update this to be IC branding?

Reply:
yep sure, forgot sorry.